src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -178,6 +184,24 @@ def load_pytorch_weights_in_tf2_model(tf_model, pt_state_dict, tf_inputs=None, a
 
         array = pt_state_dict[name].numpy()
 
+        if name.endswith("depthwise.weight"):
+            array = numpy.transpose(array, axes=(2, 0, 1))
+            transpose = False
+
+        if name.endswith("pointwise.weight"):
+            array = numpy.transpose(array, axes=(2, 1, 0))
+            transpose = False
+
+        if name.endswith("conv_attn_key.bias"):
+            array = numpy.squeeze(array)
+            transpose = False
+
+        if 1 == 0:
+            if name.endswith("intermediate.dense.weight") or name.endswith("output.dense.weight"):
+                if not name.endswith("attention.output.dense.weight"):
+                    print(name)
+                    transpose = False
```

Comment:
Does `medium` or `small` appears in the name? If yes we can use the full name of the model to activate/deactivate this branch.

Also I don't get this double `if` condition, why not simply use the second `if`? Does the `attention.output.dense` layer appears only in the `small` or `medium` version of the model?